### DISCUSSION TITLE
Dark Mode or where is it?

### DISCUSSION THREAD

**Discussion by gorkmbs** [npm/feedback#1055] [Website] [answered]
The heaven themed website needs a dark mode. We are humans. I like my eyes. Please add a dark mode. If you already have it??? Where is it. Please put it in a visible place. Come on. We are all developers. 

Best regards.

**Comment by ljharb** [comment]
Duplicate of #420, and not all developers prefer dark mode.

**Comment by gorkmbs** [comment, marked as answer]
Another one is opened in 2021. And after years, we are in 2024. Still no dark mode! Maybe it is time to start... I have a big screen and I dont want my screen to be a led light at night. Please reconsider your decision and add a dark mode. We dont want a perfect one. Just a simple dark mode please.
